--- mingw-std-threads-ee67ef384470e998c8e0b7301f7a79b5019251a2/mingw.shared_mutex.h.orig	2018-09-11 17:06:14.000000000 +0200
+++ mingw-std-threads-ee67ef384470e998c8e0b7301f7a79b5019251a2/mingw.shared_mutex.h	2018-10-04 20:56:12.195504727 +0200
@@ -32,6 +32,7 @@
 #endif
 
 #include <cassert>
+#include <climits>
 
 //    Use MinGW's shared_lock class template, if it's available. Requires C++14.
 //  If unavailable (eg. because this library is being used in C++11), then an
